.github/workflows/add-issue.yml

```diff
@@ -0,0 +1,16 @@
+name: Add Issue to Board
+
+on:
+  issues:
+    types: [opened]
+
+jobs:
+  add:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: alex-page/github-project-automation-plus@v0.8.1
+        with:
+          project: Tweets
+          column: Backlog
+          repo-token: ${{ secrets.GHPROJECT_TOKEN }}
```

Comment:
huh, why is this not just `secrets.GITHUB_TOKEN`?

Reply:
Because that action has way too many options including org projects. I didn't fully figure out why it needs a PAT, but my guess would be that. I didn't find another action we could use so far, but I feel down the road we can just do our own very simple one. Will see. This is definitely not optimal.

Reply:
Do we even need it? I think I was intending on using the automation that's built into projects:
![image](https://user-images.githubusercontent.com/640949/165737997-c8e13310-6723-444d-863f-f43cfb1c0695.png)

Reply:
As far as I've seen that working, this only adds it to the column when adding to the project. It doesn't automatically add new issues to the project itself. Or did I miss something?

Reply:
Ah, that's possible, I never got around actually trying it.